spec/system/visit_sign_in_spec.rb

```diff
@@ -164,7 +165,7 @@
     sign_in(user.email, user.password)
     expect(current_path).to eql new_user_session_path
     expect(page).to have_content t('mfa_enrolment.heading')
-    expect(page).to have_selector(".mfa-qr-code svg")
+    expect(page).to have_css("div#QRCode")
```

Comment:
we should include the svg tag as well if possible
```suggestion
    expect(page).to have_css("#qr-code svg")
```

Reply:
Made the changes, I had to hyphenate the entire id though i.e  qr-code-svg.

Reply:
the svg should not be part of the ID... svg is an element on its own, so the suggested`"#qr-code svg"` will be looking for `<element id="qr-code"> ... <svg>` whereas your change is not going to check whether there's any svg element